rules/rbe_repo.bzl

```diff
@@ -319,33 +325,39 @@ def _validate_host(ctx):
     if not ctx.which("tar"):
         fail("Cannot run rbe_autoconfig as 'tar' was not found on the path.")
 
-# Produces BUILD files with aliases for all the required toolchain / platform targets.
-def _use_standard_config(ctx, bazel_version, config_version):
+# Produces BUILD files with alias for the C++/Java toolchain targets.
+def _use_standard_config(ctx):
     print("Using checked-in configs.")
 
-    # Create the BUILD files with the aliases
-    template = ctx.path(Label("@bazel_toolchains//rules:BUILD.std_container.tpl"))
-    jdk = "@bazel_toolchains//configs/ubuntu16_04_clang/%s:jdk8" % config_version
-    cc_toolchain = "@bazel_toolchains//configs/ubuntu16_04_clang/%s/bazel_%s/cpp:cc-toolchain-clang-x86_64-default" % (config_version, bazel_version)
-    platform = "@bazel_toolchains//configs/ubuntu16_04_clang/%s:rbe_ubuntu1604" % config_version
+    # Create the BUILD file with the alias for the cc_toolchain_suite
+    template = ctx.path(Label("@bazel_toolchains//rules:BUILD.std_cc_toolchain.tpl"))
```

Comment:
use std_cc_toolchain is confusing, maybe BUILD.cc_alias.tpl

Reply:
same thing to the java one

Reply:
fixed